src/util/snapshots.tsx

```diff
@@ -126,7 +109,9 @@ export const getSnapshotSchema = (
       .test(
         ...testDuplicateSnapshotName(instance, controllerState, snapshotName),
       )
-      .test(...testForbiddenChars()),
+      .matches(/^[A-Za-z0-9-_.:]+$/, {
```

Comment:
This is validating only the snapshot schema, hence for manual snapshot creation/edit only. Ideally, we should validate also the "Snapshot name pattern" config field. Otherwise we're solving only half of the problem - or even less, if we consider that scheduled snapshots are much more likely to be used than manual ones.

Reply:
The automatic pattern is not easily verifiable. The pogo2 template needs to be parsed and the resulting string put into this validator. But we don't have a parser at hand. So I'd keep that part untouched and fix the manual snapshot naming validation as reported in #334

Reply:
I guess that makes sense... at least until someone else opens an issue about the snapshot name pattern not being validated :)